src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,119 @@
+.contacts{
+    background-color: $pageBg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items:center;
+    
+
+
+&__head{
+    display:none;
+}
+    
+&__box{ 
+    display:flex;
+    flex-direction: column;
+    gap:20px;
+    max-width:220px;
```

Comment:
wydaje mi się, że możesz dać więcej, np. 480px tak aby te całe kafelki się rozszerzały do tableta. Może nie aż do 768, bo już nie ładnie to wygląda, ale 480 jest ok. Jakbyś zostawił to tak jak teraz to dla szerokości 760px kafelki są strasznie wąskie, za wąskie wg mnie :)